packages/core/src/external/sftp/types.ts

```
@@ -0,0 +1,89 @@
+  | SftpCloneAction;
+
+export interface SftpBaseAction {
+  type: "read" | "write" | "list" | "exists" | "clone";
+}
+
+export type SftpResult<A extends SftpBaseAction> = SftpBaseAction extends A
+  ? unknown
+  : A extends { type: "read" }
+  ? Buffer
+  : A extends { type: "write" }
+  ? void
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)